### DISCUSSION TITLE
перестал работать на MacOS

### DISCUSSION THREAD

**Discussion by AFoXi** [xvzc/SpoofDPI#254] [General]
месяц отработал "на ура", сегодня перестал работать.  для  GoodbyeDPI уже выкатили решение, ждать ли решения от SpoofDPI? заранее спасибо.

**Comment by Fervex1983** [comment]
Аналогичная проблема. С утра перестал работать.

**Comment by nrvnm** [comment]
Тоже не работает. Может провайдер анти меры какие-то принял.. Фикс будет?

**Comment by prontok** [comment]
На Linux тоже перестал работать. Видимо действительно провайдер что то делает

**Comment by ghost** [comment]
Привет всем нищим у которых нет 300р на vds.
Тоже не работает.

**Reply by AFoXi** [reply]
порекомендуете что-то конкретное за 300 руб.? 🤔

**Reply by ghost** [reply]
Можно даже за 100р, не на всех российских vds dpi блокировки, только региональные от самих ресурсов на российские ip.
